ADALiOS/ADALiOS/ADHelpers.m

```diff
@@ -245,5 +246,39 @@ +(uint8_t*) updateDataInput:(uint8_t) ctr
     return tmpFixedInput;
 }
 
++ (NSURL*)addClientVersionToURL:(NSURL*)url
+{
+    if (!url)
```

Comment:
Are you saying you'd prefer to see:
if (!url) return nil;
all on a single line?

When it's just simple error-check/return I prefer to keep it to two lines, otherwise they can start to crowd out the business logic code